src/sql/azdata.proposed.d.ts

```diff
@@ -604,6 +605,22 @@ declare module 'azdata' {
 		propertyItems?: PropertiesContainerItem[];
 	}
 
+	/**
+	 * Component to display text with an icon representing the severity
+	 */
+	export interface InfoBoxComponent extends Component, InfoBoxComponentProperties {
+	}
+
+	export type InfoBoxStyle = 'information' | 'warning' | 'error' | 'success';
```

Comment:
Maybe `InfoBoxType`? Style to me is generally more about the CSS styling - which is a side-effect of the different types of information boxes that can be displayed

Reply:
I am trying to use the same name as the portal, i started with type and changed to style.

![image](https://user-images.githubusercontent.com/13777222/105554604-8894eb00-5cbc-11eb-9690-43457ff120b4.png)